app/_hub/kong-inc/mocking/index.md

```diff
@@ -401,6 +401,25 @@ curl -X POST http://<admin-hostname>:8001/routes/getStockQuote/plugins \
     --data "config.api_specification_filename=stock-0.1.json"
 ```
 
+Optional configuration for random delay:
+
+```
+curl -X POST http://<admin-hostname>:8001/routes/getStockQuote/plugins \
+    --data "name=mocking"  \
+    --data "config.api_specification_filename=stock-0.1.json" \
+    --data "config.random_delay=true" \
+    --data "config.max_delay_time=1" \
+    --data "config.min_delay_time=0.001"
+```
+
+DB-less configuration:
+
+```
+curl -X POST http://<admin-hostname>:8001/routes/getStockQuote/plugins \
+    --data "name=mocking"  \
+    --data "config.api_specification=stock-0.1.json"
```

Comment:
note the different config-not api spec filename

Reply:
After Shane's demo, this is wrong. You paste the entire spec in here. This would be difficult to display gracefully.